### PR TITLE
add value to living

### PR DESCRIPTION
**Pull request by Studgi**

### CODE REVIEW

File: living/living.go [hunk 1]
```diff
@@ -41,6 +42,8 @@ type Living struct {
 
 	h  atomic.Value[Handler]
 	mc *entity.MovementComputer
+
+	data sync.Map
```
Comment (RestartFU) [inline living/living.go:46]: can you rename this to values

File: living/living.go [hunk 2]
```diff
@@ -460,3 +463,14 @@ func (e *Living) viewers() []world.Viewer {
...
+
+// Stores value at runtime. To store it even after server restarts, you'll need to encode/decode NBT.
+func (e *Living) WithValue(key string, value any) {
+	e.data.Store(key, value)
+}
+
+// Retrieves value at runtime. To store it even after server restarts, you'll need to encode/decode NBT. Returns the value and whether it exists.
+func (e *Living) Value(key string) (any, bool) {
+	value, exists := e.data.Load(key)
+	return value, exists
```
Comment (RestartFU) [inline living/living.go:475]: here just return e.values.Load(key) please

**Comment by RestartFU** [comment]
once the comments are fixed I'll merge

**Review by RestartFU — CHANGES_REQUESTED** [review]

**Comment by Studgi** [comment]
Ok will do later

**Pull request closed without merging**